### PR TITLE
Assign build-time baseUrl in page template

### PR DESCRIPTION
**Pull request by nicholaschuayunzhi**
`baseUrl` is recorded based on build time baseUrl.

This ensures that `baseUrl` is the one author specifies.
- using baseUrl in site.json 
- using the `--baseUrl` flag in `markbind build`

Thanks @rachx for suggesting solution.

### CODE REVIEW

**Comment by nicholaschuayunzhi** [comment]
I have tested here: https://nicholaschuayunzhi.github.io/website-1/
Please note that i had to change my baseUrl to website-1 because i have to repositories named website 😅

File: lib/template/page.ejs [hunk 1]
```diff
@@ -16,5 +16,6 @@
 </body>
 <script src="<%- asset.vue %>"></script>
 <script src="<%- asset.vueStrap %>"></script>
+<script>const baseUrl = '<%= baseUrl %>'</script>
```
Comment (acjh) [inline lib/template/page.ejs]: Use block to differentiate this:

```html
<script>
    const baseUrl = '<%- baseUrl %>';
</script>
```

If we don't need to escape it (`<%=`), let's keep it consistent (`<%-`).

Comment (acjh) [reply lib/template/page.ejs]: Trailing semicolon!

**Comment by acjh** [comment]
@nicholaschuayunzhi You can merge this :)

**Pull request merged**